sdk/docs/core/CHANGELOG.md

```diff
@@ -4,8 +4,22 @@
 ## 1.0.0-preview.4 (Unreleased)
 
 
-## 1.0.0-preview.3 (Unreleased)
+## 1.0.0-preview.3 (2020-07-20)
 
+- Updated `az_result` values:
+  - Rename `az_result` value `AZ_ERROR_PARSER_UNEXPECTED_CHAR` to `AZ_ERROR_UNEXPECTED_CHAR`.
+  - Remove unused `az_result` error codes: `AZ_ERROR_JSON_STRING_END`, `AZ_ERROR_JSON_POINTER_TOKEN_END`, and `AZ_ERROR_MUTEX`.
+- Add permutations of numeric type parsing and formatting APIs on span - ato[u|i][32|64], atod and the inverse [u|i][32|64]toa, dtoa.
+- Updates to the JSON APIs:
+  - Rename JSON parser/builder APIs to reader/writer.
+  - Add double parsing and formatting support to JSON reader and JSON writer.
+  - Redesign JSON reader and JSON token APIs with lazy evaluation of tokens, proper unescaping support, and hardened validation.
+- Update samples, README docs along with deep dive video, and VSCode and CMake instructions.
```

Comment:
Can you add a line here with
```
- Add PnP sample for Azure IoT Hub
```

Reply:
Was that the only sample you want called out explicitly? I saw a whole bunch of sample updates, moves and changes.

Reply:
Sample updates were done but PnP was the only new one added.